tests/validation/tests/v3_api/test_cli.py

```diff
@@ -114,6 +119,16 @@ def test_cli_app_install(remove_cli_resource, rancher_cli: RancherCli):
     assert initial_app["version"] == "1.5.0"
 
 
+def test_cli_app_values_install(remove_cli_resource, rancher_cli: RancherCli):
+    rancher_cli.log.info("Testing Upgrading Apps")
+    initial_app = rancher_cli.apps.install(
+        "chartmuseum", random_str(), version="2.3.1", timeout=120,
```

Comment:
Can we have all the chart versions on the top as a global variable? This would make it easier when/if the version changes to change it at one place? 
Could we also have the timeouts also declared as global variables?